updatecli/updatecli.d/update-k8s-k3s/autodiscovery-k8s-deps.yaml

```diff
@@ -0,0 +1,23 @@
+name: '{{ .k8s_k3s.manifest }}'
+pipelineid: '{{ .k8s_k3s.pipelineid }}'
+
+autodiscovery:
+  scmid: 'default'
+  actionid: 'rancher-pr'
+  groupby: all
+  crawlers:
+    golang:
+      versionfilter:
+        kind: semver
+        pattern: '{{ .k8s.deps }}'
+      only:
+        - modules:
+            '^k8s\.io\/.*': ''
```

Comment:
Please drop from renovate the dependencies this will be handling, to avoid duplicated PRs:
```
  "packageRules": [
    {
      "matchPackageNames": ["k8s.io/*"],
      "matchBranch": "main",
      "enabled": false
    }
  ]
```